application/backend/src/main/java/io/typerefinery/websight/models/components/DefaultStyledGridComponent.java

```diff
@@ -93,12 +104,11 @@ public Integer getMdOffset() {
   public Integer getLgOffset() {
     return grid.getLgOffset();
   }
+
   @PostConstruct
   private void init() {
-    componentClasses = Stream.concat(
-            Arrays.stream(style.getClasses()),
-            new GridStyle( this).getClasses().stream())
+    componentClasses = Arrays.stream(style.getClasses())
```

Comment:
Where are the grid styles being used?

Reply:
Grid styles are used for the variant parent classes.

Because variantClassName have width of 100%, this grid class will have col-n values. 